spec/features/manage_cmr/groups/groups_list_spec.rb

```diff
@@ -62,7 +62,6 @@
               within '.groups-table' do
                 expect(page).to have_no_content('Administrators CMR Administrators CMR 2')
                 expect(page).to have_no_content('Administrators_2 The group of users that manages the CMR. CMR')
-                expect(page).to have_no_content('SEDAC Admin Group Test group for provider SEDAC')
```

Comment:
Is this test failing? The way the ACLs for the group are set up, I think that only the admin user can see this group, which is why it's grouped with the system groups for the test

Reply:
Reverted.